### PR TITLE
DOMA-3822 add organization in Message creation

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**

### CODE REVIEW

File: apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js [hunk 1]
```diff
@@ -80,6 +81,7 @@ const prepareAndSendNotification = async (context, receipt, resident) => {
         meta: { dv: 1, data },
         sender: { dv: 1, fingerprint: 'send-billing-receipts-added-notifications' },
         uniqKey: notificationKey,
+        organization: organizationId && { connect: { id: organizationId } },
```
Comment (akarjakin) [inline apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js:84]: `organization: organizationId && { connect: { id: resident.organization.id } },`

File: apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js [hunk 2]
```diff
@@ -141,11 +143,12 @@ const sendBillingReceiptsAddedNotificationsForPeriod = async (receiptsWhere, onL
 
             for (const consumer of consumers) {
                 const resident = get(consumer, 'resident')
+                const organizationId = get(consumer, ['organization', 'id'])
 
                 // ServiceConsumer has no connection to Resident
                 if (!resident) continue
 
-                const success = await prepareAndSendNotification(context, receipt, resident)
+                const success = await prepareAndSendNotification(context, receipt, resident, organizationId)
```
Comment (akarjakin) [inline apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js:151]: `organizationId` is redundant

**Review by akarjakin — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/notification/schema/SendMessageService.js [hunk 3]
```diff
@@ -93,7 +93,7 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
         },
         {
             access: true,
-            type: 'input SendMessageInput { dv: Int!, sender: SenderFieldInput!, to: SendMessageToInput!, emailFrom: String, type: SendMessageType!, lang: SendMessageLang!, meta: JSON!, organization: OrganizationWhereUniqueInput, uniqKey: String }',
+            type: 'input SendMessageInput { dv: Int!, sender: SenderFieldInput!, to: SendMessageToInput!, emailFrom: String, type: SendMessageType!, lang: SendMessageLang!, meta: JSON!, organization: OrganizationRelateToOneInput, uniqKey: String }',
```
Comment (pahaz) [inline apps/condo/domains/notification/schema/SendMessageService.js:96]: Why you need to change the type?

File: apps/condo/domains/notification/schema/SendMessageService.js [hunk 4]
```diff
@@ -120,14 +120,14 @@ const SendMessageService = new GQLCustomSchema('SendMessageService', {
             resolver: async (parent, args, context) => {
                 // TODO(pahaz): think about sending emails with attachments
                 const { data } = args
-                const { dv, sender, to, emailFrom, type, meta, lang, uniqKey } = data
+                const { dv, sender, to, emailFrom, type, meta, lang, uniqKey, organization } = data
```
Comment (pahaz) [inline apps/condo/domains/notification/schema/SendMessageService.js:123]: It’s better to check the access to org

File: apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js [hunk 5]
```diff
@@ -46,6 +46,7 @@ const getMessageTypeAndDebt = (toPay, toPayCharge) => {
  * @param keystone
  * @param receipt
  * @param resident
+ * @param organizationId
```
Comment (pahaz) [inline apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js:49]: You forgot to add the arg inside function

Comment (pahaz) [reply apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotifications.js:49]: It looks redundant

File: apps/condo/jest.setupSpec.js [hunk 6]
```diff
@@ -2,6 +2,8 @@ const falsey = require('falsey')
 
 const EXTRA_LOGGING = falsey(process.env.DISABLE_LOGGING)
 
+jest.setTimeout(60000)
```
Comment (pahaz) [inline apps/condo/jest.setupSpec.js:5]: Use argument for jest locally like inside CI

File: apps/condo/schema.ts [hunk 7]
```diff
@@ -35591,7 +35591,7 @@ export type SendMessageInput = {
   type: SendMessageType;
   lang: SendMessageLang;
   meta: Scalars['JSON'];
-  organization?: Maybe<OrganizationWhereUniqueInput>;
+  organization?: Maybe<OrganizationRelateToOneInput>;
```
Comment (pahaz) [inline apps/condo/schema.ts:35594]: Why?

File: apps/condo/domains/notification/schema/SendMessageService.js (hunk 3 above)
Comment (nomerdvadcatpyat) [reply apps/condo/domains/notification/schema/SendMessageService.js:96]: returned back

**Pull request merged**